packages/devextreme/js/viz/series/points/range_bar_point.js

```diff
@@ -9,6 +9,10 @@ export default _extend({}, barPoint, {
 
     _getFormatObject: rangeSymbolPointMethods._getFormatObject,
 
+    _calculateVisibility: function(x, y, width, height) {
+        this.inVisibleArea = barPoint._getPointVisibility.call(this, x, y, width, height, false);
```

Comment:
Seems like we can just override it, no?

Reply:
override with the same function but with `>` instead of `>=`? We can, but current way seems more clear